File: File cpp/Kalkulator.cpp
#include <iostream>
#include <string>

using namespace std;

int main(){
    float a,b,hasil;
    int pilih;

    cout<<"Program Kalkulator"<<endl;

    cout<<"1. Tambah"<<endl;
    cout<<"2. Kurang"<<endl;
    cout<<"3. Kali"<<endl;
    cout<<"4. Bagi"<<endl;

    cout<<"Masukkan Bilangan Pertama : "; cin>>a;
    cout<<"Masukkan Bilangan Kedua : "; cin>>b;
    cout<<"Pilih Operasi aritmatika yang akan digunakan : "; cin>>pilih;

    switch(pilih){
        case 1:
            hasil= a + b;
            break;
        case 2:
            hasil = a - b;
            break;
        case 3:
            hasil = a * b;
            break;
        case 4: 
            hasil = a / b;
            break;
}

    cout<<"Hasilnya adalah : "<<hasil<<endl;

    system("pause");
    return 0;
}

File: File py/Kalkulator.py
def hitung(a, b, pilih):
    match pilih:
        case 1:
            hasil = a + b
            return hasil
        case 2:
            hasil = a - b
            return hasil
        case 3:
            hasil = a * b
            return hasil
        case 4:
            hasil = a/b
            return hasil        

print("Program Kalkulator Sederhana")
print("1. Tambah")
print("2. Kurang")
print("3. Kali")
print("4. Bagi\n")

a = float(input("Masukkan bilangan pertama : "))
b = float(input("Masukkan bilangan kedua : "))

pilih = int(input("Pilih Operasi aritmatika yang akan digunakan : "))

print("Hasilnya adalah : ", hitung(a, b, pilih))
